### PR TITLE
recipe view button/modal

### PR DESCRIPTION
**Pull request by tiftran**

### CODE REVIEW

**Review by rehandalal — APPROVED** [review]
nice work! thanks for this

bors r+

File: extension/content/components/pages/RecipesPage.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import autobind from "autobind-decorator";
 import React from "react";
 import { Controlled as CodeMirror } from "react-codemirror2";
+import Highlight from "react-highlight";
```
Comment (rehandalal) [inline extension/content/components/pages/RecipesPage.js]: oops! this should be:
```js
import Highlight from "devtools/components/common/Highlight";
```

**Review by rehandalal — CHANGES_REQUESTED** [review]
needs an import path updated

**Review by mythmon — DISMISSED** [review]
Looks like you added a new dependency, `react-highlight`, but didn't add it to `package.json`? If you're using Yarn, you can say `yarn add react-highlight`.

**Comment by rehandalal** [comment]
> Looks like you added a new dependency, react-highlight, but didn't add it to package.json? If you're using Yarn, you can say yarn add react-highlight.

`react-highlight` was a dependency but was removed in #27 since it was breaking tree-shaking

File: extension/content/components/pages/RecipesPage.js [hunk 2]
```diff
@@ -197,6 +198,7 @@ class RecipesPage extends React.PureComponent {
       this.setState({ runningArbitrary: false });
     }
   }
+  /* eslint-enable no-useless-catch */
```
Comment (rehandalal) [inline extension/content/components/pages/RecipesPage.js]: this should actually go up above on line 200 above the last `}`.

Comment (tiftran) [reply extension/content/components/pages/RecipesPage.js]: will do

**Review by rehandalal — APPROVED** [review]
bors r+

**Pull request merged**